### DISCUSSION TITLE
Displaying jobs to assigned user

### DISCUSSION THREAD

**Discussion by keshavadk** [cvat-ai/cvat#4009] [Q&A]
In CVAT task page, Can we assign jobs to different users under one task ? .
For example: One task will be having 3 jobs. I have attached the images below.
So assign job1 and job2 to user1 , then assign job3 to user2. Tried assigning the jobs to user1 and user2 , but in the "Assigned to" field ,it is user1. But when I login as user1, all the jobs that are job1, job2 and job3 will be visible. But i was expecting only job1 and job2 should be visible to user1.

Thank you. 

![1](https://user-images.githubusercontent.com/16021947/145387697-df4b1371-d585-40da-a9b4-8efba645287c.JPG)
![2](https://user-images.githubusercontent.com/16021947/145387705-2dabc94c-5f7a-4c10-a21b-ad3eaaa74089.JPG)

**Comment by nmanovic** [comment]
It will help when it is merged: https://github.com/openvinotoolkit/cvat/pull/3788

**Reply by keshva23** [reply]
@nmanovic  Ok, Thank you.

But Can i know , how can we assign single task to multiple users.

**Comment by nmanovic** [comment]
@keshavadk , if you really need it, you can do that in context of organizations. Maintainers of an organization will be able to see all tasks in the organization. As previously it will be possible to set an assignee for a task and assignees for specific jobs.

**Comment by keshavadk** [comment]
@nmanovic <https://github.com/nmanovic> Ok, Thank you.
